service/topology_state_machine.hh

```diff
@@ -195,6 +195,13 @@ struct topology {
     // The set of nodes that should be considered dead during topology operations
     std::unordered_set<raft::server_id> ignored_nodes;
 
+    // The set of nodes currently excluded from synchronization in the tablets management code.
+    // The barrier should not wait for these nodes.
+    // This set is effectively equal to: ignored_nodes + keys(left_nodes_rs).
+    // Tablet replicas may temporarily include left nodes (e.g. when a node is replaced),
+    // hence the need for this field.
```

Comment:
This comment is confusing because the code related to vnodes also uses it. Perhaps it's made clear in the following commits.